configure.ac

```diff
@@ -69,6 +69,13 @@ x86_64*)
   ;;
   esac
 ;;
+*64*)
+;;
+*)
+AC_MSG_WARN([
+*** __BIGREGION__ is not defined.
+*** Adjust the address limits in sasconf.h as required.])
+;;
```

Comment:
I still believe we should define __BIGREGION__ automatically based on the information provided by AC_CANONICAL_BUILD.  It shouldn't be complex and we won't be exposing SPHDE internals to the users.